pkg/remote/codec/header_codec.go

```diff
@@ -253,6 +253,26 @@ func writeKVInfo(writtenSize int, message remote.Message, out remote.ByteBuffer)
 			writeSize = writeSize + 2 + valWLen
 		}
 	}
+
+	// custom kv info
+	customMap := tm.TransCustomInfo()
+	customStr, err := utils.Map2JSONStr(customMap)
```

Comment:
Why should this be converted to a json string, the correct value should be gdpr-token